lib/cs_guide/resources/venue.ex

```diff
@@ -9,8 +9,14 @@ defmodule CsGuide.Resources.Venue do
 
   schema "venues" do
     field(:venue_name, :string)
+    field(:description, :string)
```

Comment:
description should be:
```
field(:description, Fields.description_plaintext_unlimited)
```
please see: https://github.com/dwyl/fields/issues/20
We _must_ do this in a modular way, or not at all.
All "free text" fields should use the sanitisation functions: https://github.com/dwyl/fields/issues/19
Otherwise we are _inviting_ people to Hack this site using XSS.